server_side/interpreter/tree_objects/select.py

```diff
@@ -0,0 +1,90 @@
+from server_side.interpreter.tree_objects.executable_tree import ExecutableTree
+
+
+class Select(ExecutableTree):
+    """
```

Comment:
Wow nice essay😁

Reply:
You're right. It's safer to merge this branch later.

Thank you, it's a good essay indeed :))